README.md

```diff
@@ -1,2 +1,8 @@
 # data-api
 Micro-service for retirieval of Hypothesis engine computed data
```

Comment:
Please change this from Micro-service to something that conveys the fact that -- This is a (npm dependency) library that defines apis instantiated by data-api-image created for the benefits of separation of concerns of build aspects from code